models/users.js

```diff
@@ -28,20 +28,22 @@ const addOrUpdate = async (userData, userId = null) => {
         })
       }
 
-      return { isNewUser, userId }
+      return { incompleteUserDetails, userId }
     }
 
     // userId is null, Add or Update user
     const user = await userModel.where('github_id', '==', userData.github_id).limit(1).get()
     if (!user.empty) {
+      userData.incompleteUserDetails = false
```

Comment:
This flag can only be false when user updates his/her `id`.
We should change this flag until the `id` is updated.